docs/platform/howto/use-aws-privatelinks.rst

```diff
@@ -0,0 +1,205 @@
+Using AWS PrivateLink with Aiven services
+=========================================
+
+AWS `PrivateLink <https://aws.amazon.com/privatelink/>`__ brings Aiven
+services to the selected virtual private cloud (VPC) in your AWS
+account. In a traditional setup that uses `VPC
+peering <https://help.aiven.io/en/articles/778836-using-vpc-peering>`__
+, traffic is routed through an AWS VPC peering connection to your Aiven
+services. With PrivateLink, you can create a VPC endpoint to your own
+VPC and access an Aiven service from that. The VPC endpoint creates
+network interfaces (NIC) to the sub-nets and availability zones that you
+choose and receives the private IP addresses that belong to the IP range
+of your VPC. The VPC endpoint is routed to your Aiven service located in
+one of Aiven's AWS accounts.
+
+You can enable PrivateLink for Aiven services located in project VPCs.
+Before you can set up AWS PrivateLink, `create a
+VPC <https://help.aiven.io/en/articles/778836>`__ and launch the
+services that you want to connect to that VPC. As there is no network
+routing between the VPCs, you can use any private IP range for the VPC,
+unless you also want to connect to the project VPC using VPC peering
+connections. This means that overlaps in the IP range are not an issue.
+
+You can use either the `Aiven web console <https://console.aiven.io>`__
+or the `Aiven CLI <https://github.com/aiven/aiven-client>`__ to set up
+AWS PrivateLink. You also need the AWS CLI to create a VPC endpoint.
```

Comment:
Is AWS CLI the only way to create a VPC endpoint? Can they use [VPC Wizard](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/gsg_create_vpc.html) from AWS console?

Reply:
The VPC wizard would be helpful to create the VPC, this point is referring to the VPC endpoint which is the interface created on the customer subnet to get connected to the Private link, yes it can be created from console: by following similar step listed in [Connect to an endpoint service as the service consumer 
](https://docs.aws.amazon.com/vpc/latest/privatelink/create-endpoint-service.html#share-endpoint-service ) 
I think we can mentioned that this could be done in the console, though I am not sure if sharing the link would cause some confusion.